nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -199,10 +206,22 @@ def reshape_bias_shift(self, bias_shift: TTensor, bias_value: TTensor, channel_a
         if bias_value.ndim > 1:
             new_shape = [1] * bias_value.ndim
             new_shape[channel_axis] = bias_shift.shape[0]
-            bias_shift = self._backend_entity.reshape_tensor(bias_shift, new_shape)
+            bias_shift = bias_shift.reshape(*new_shape)
         return bias_shift
 
-    def _get_fp_inputs(self, statistic_points: StatisticPointsContainer, node_name: str) -> Tuple[List, List]:
+    @staticmethod
+    def _get_bias_shift_magnitude(current_bias_value: NNCFTensor, updated_bias_value: NNCFTensor) -> float:
```

Comment:
Once proper `NNCFTensor`s are passed into the algorithm, the code involving the tensor manipulations can be written once and be working out of the box for all backends (i.e. backend-specific implementations of `NNCFTensor`). The ex-backend-specific methods for the same were straightforward to move into the common algo part.